rails/embedded_analytics/README.md

```diff
@@ -1,24 +1,30 @@
-# README
+# Rails app
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+This Rails application demonstrates a simple, barebones Metabase dashboard embedded in a web application.
 
-Things you may want to cover:
+## Prerequisites
 
-* Ruby version
+- **Metabase**. You should have already completed the setup detailed in the [README](../README.md) for this repository, which shows you how to get an instance of Metabase up and running in the [metabase](../metabase) directory of this repository.
 
-* System dependencies
+- **Rails**. You'll need to have [Ruby on Rails installed](https://guides.rubyonrails.org/getting_started.html) on your machine.
```

Comment:
Similar to Python, the `bundle install` will install Rails, so I think we just need to say they should have ruby and bundler installed.